spec/features/users_can_view_an_activitys_details_spec.rb

```diff
@@ -17,7 +17,7 @@
         expect(page).to have_content "Details"
       end
       expect(page).to have_content activity.title
-      expect(page).to have_link t("page_content.activity.implementing_organisation.button.new")
+      expect(page).to have_content t("page_content.activity.implementing_organisation.heading")
```

Comment:
Has this slipped in from another commit?

Reply:
Nope! This spec doesn't set up an editable report, and after fixing the policy checks, implem orgs cannot be added while there's no editable report.